packages/replay-next/src/suspense/NetworkRequestsCache.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  RequestBodyData,
+  RequestEventInfo,
+  RequestInfo,
+  ResponseBodyData,
+} from "@replayio/protocol";
+import { createCache } from "suspense";
+
+import { compareExecutionPoints } from "replay-next/src/utils/time";
+import { ReplayClientInterface } from "shared/client/types";
+
+export type NetworkRequestsCacheData = {
+  requestEventInfo: RequestEventInfo[];
+  requestInfo: RequestInfo[];
+  responseBodyData: ResponseBodyData[];
+  requestBodyData: RequestBodyData[];
+};
+
+export const networkRequestsCache = createCache<
+  [replayClient: ReplayClientInterface],
+  NetworkRequestsCacheData
+>({
+  debugLabel: "NetworkRequestsCache",
```

Comment:
I needed network requests to be loaded into a Suspense cache so that I could fetch them in the test migration code. I've filed a follow up Linear issue to migrate our Network panel over to use this cache as well, and we can remove the redundant network data in Redux.